lib/models/candle/audit_gaps.js

```diff
@@ -0,0 +1,26 @@
+const { TIME_FRAME_WIDTHS } = require('bfx-hf-util')
+const _isFinite = require('lodash/isFinite')
+
+module.exports = ({ getInRange }) => (doc, { start, end }) => {
+  const gaps = []
+  const { tf } = doc
+  const candles = getInRange(doc, { start, end })
+
+  if (candles.length < 2) {
+    return { gaps, candles }
```

Comment:
Seems stratnge to return empty gaps here? If the candles returned are less than 2 maybe we should just consider the whole thing a gap?

i.e:

```
if (candles.length < 2) {
    gaps.push({start, end})
    return { gaps, candles: [] }
}
```

Reply:
I was initially going to include this, but due to my comment on your other remark I'm keeping it out; if there are no candles, there are no gaps by candle width (this method is meant to check for consistency, not presence.)